packages/spacebars-compiler/codegen.js

```diff
@@ -27,7 +27,9 @@ var builtInTemplateMacros = {
   // Confusingly, this makes `{{> Template.dynamic}}` an alias
   // for `{{> __dynamic}}`, where "__dynamic" is the template that
   // implements the dynamic template feature.
-  'dynamic': 'Template.__dynamic'
+  'dynamic': 'Template.__dynamic',
+
+  'ready': 'view.templateInstance().ready()'
```

Comment:
If `view` isn't a template, this will throw an error that `view.templateInstance` is undefined, right?  Shouldn't we throw a better error?